src/rgw/rgw_file.cc

```diff
@@ -1145,8 +1145,8 @@ namespace rgw {
       emplace_attr(RGW_ATTR_SLO_UINDICATOR, std::move(slo_userindicator_bl));
     }
 
-    op_ret = processor->complete(s->obj_size, etag, &mtime, real_time(), attrs, delete_at,
-				 if_match, if_nomatch);
+    op_ret = processor->complete(s->obj_size, etag, &mtime, real_time(), attrs,
+                                 (delete_at ? *delete_at : real_time()), if_match, if_nomatch);
```

Comment:
If `delete_at` is empty, then we supply an instance of `real_time`. `ceph::real_time` is an alias over `std::chrono::time_point<ceph::time_detail::real_clock>`. Its default-constructed instance represents *logical zero*  --`real_clock::is_zero(ceph::real_time()) == true`. OK.